projects/js-packages/components/components/threats-data-views/index.tsx

```diff
@@ -80,10 +79,6 @@ export default function ThreatsDataViews( {
 	onUnignoreThreats?: ActionButton< Threat >[ 'callback' ];
 } ): JSX.Element {
 	const baseView = {
-		sort: {
-			field: 'severity',
-			direction: 'desc' as SortDirection,
-		},
```

Comment:
Since we are removing this, could we allow consumers to provide a default sort?

i.e. Free Protect would not provide, paid Protect would pass in the severity sort alongside the "active" filter by default.

Reply:
>Since we are removing this, could we allow consumers to provide a default sort?

If we keep with the custom sorting (or server side), I think providing a sort property would render it useless as it would take that sorted list and resort it.